hrms/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -725,14 +725,14 @@ def add_earning_for_hourly_wages(self, doc, salary_component, amount):
 			}
 			doc.append("earnings", wages_row)
 
-	def set_salary_structure_assignment(self):
+	def set_salary_structure_assignment(self,docstatus=1):
```

Comment:
can we also allow passing any custom salary structure assignment?
Just passing in draft would not allow use the preview salary slip for a particular salary structure assignment

Reply:
Added